.babelrc

```diff
@@ -6,16 +6,52 @@
     "transform-class-properties",
     "array-includes",
     [
-        "transform-runtime", {
-            "polyfill": false,
-            "regenerator": true
-        }
+      "transform-runtime", {
+        "polyfill": false,
+        "regenerator": true
+      }
     ]
   ],
   "presets": [
-	  "flow",
-	  "es2016",
-	  "es2017",
-	  "react"
-  ]
+    "flow",
+    "react",
+    ["env", {
+      "targets": {
+        "node": "current"
+      }
+    }]
+  ],
+  "env": {
+    "browser": {
+      "plugins": [
+        "lodash",
+        "transform-es2015-modules-commonjs",
+        "transform-object-rest-spread",
+        "transform-class-properties",
+        "array-includes",
+        [
+          "transform-runtime", {
+            "polyfill": false,
+            "regenerator": true
+          }
+        ]
+      ],
+      "presets": [
+        "flow",
+        "react",
+        ["env", {
```

Comment:
Hmm, I think it'd be good to build some margin of error in here. I'd particularly like to make sure the site keeps working for CatCat, so let's see what version of FF he's using and try to keep code compatible with that (even if our official minimum is higher).

Reply:
I thought about this, and I believe that IE 11 provides us any margin for error we'd need, though it'd be good to get some testing.

Reply:
CatCat confirmed he's on FF 40, so let's use that instead of 44.